packages/icons/README.md

```diff
@@ -10,11 +10,12 @@ npm i pcln-icons
 
 ```js
 import React from 'react'
-import FlightsIcon from 'pcln-icons/lib/Flights'
+import { Flights as FlightsIcon } from 'pcln-icons'
 
 export default props => <FlightsIcon mr={2} />
 ```
 
+// TODO: revisit this
```

Comment:
Do we want to leave this TODO?

Reply:
Nope, I will update the documentation here. I added this stub before we were sure that `Icon` in core would be removed.

Reply:
I've pushed a commit that updates this section. Please re-review, and resolve this conversation if it is satisfactory.